src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -2702,7 +2702,8 @@ private bool CheckInArgTypes(TexlNode left, TexlNode right)
                     return false;
                 }
 
-                if (_txb.Document.Properties.EnabledFeatures.IsEnhancedDelegationEnabled && typeLeft.IsTable)
+                if ((_txb.Document == null || _txb.Document.Properties.EnabledFeatures.IsEnhancedDelegationEnabled)
```

Comment:
Not sure whether this is correct. Perhaps it should be `&&`?

Reply:
I think || is fine. 

We should assume that all feature flags will eventually become True.

Reply:
`&&` would be safer. Currently the `in` operator doesn't work for tables in the interpreter for general tables, and even on Canvas it only works for a specific type of tables (multi-select option sets).

Reply:
Yea, I feel && is better too, since its specific to a type of table that we are adding support for and only when the flag is set.

Reply:
Changed the condition.